chains/solana/contracts/programs/ccip-offramp/src/instructions/v1/execute.rs

```diff
@@ -243,9 +244,16 @@ fn internal_execute<'info>(
             accs.user_token_account.to_account_info(),
         ];
         acc_infos.extend_from_slice(accs.remaining_accounts);
+
+        let seeds = &[
+            seed::EXTERNAL_TOKEN_POOL,
+            accs.pool_program.key.as_ref(),
+            &[accs.ccip_offramp_pool_signer_bump],
```

Comment:
how does this work? Does the bump also include the actual offramp program id?

Reply:
Given that this is executed in the offramp code, the offramp program id is implicit. It's not included in the bump, that's just a `u8`, but you can only sign with a PDA that belongs to the same program, so it uses always uses the offramp program id.